content/introduction.md

```diff
@@ -29,10 +29,15 @@ Alternatively, to install the TypeScript starter project with **Git**:
 $ git clone https://github.com/nestjs/typescript-starter.git project
 $ cd project
 $ npm install
-$ npm run start
 ```
 
-Open your browser and navigate to `http://localhost:3000/`.
+To run the project, use the following command (make sure you're in the project folder):
+
+```bash
+$ nest start
```

Comment:
We should stick to `npm run start` here. `npm start` will call `nest start` either way, but it would lookup for a binary in the local `node_modules`. Hence, we won't force anyone to install `@nestjs/cli` globally.

Reply:
Makes sense.  Closing.